bucket/streamdeck.json

```
@@ -0,0 +1,33 @@
+{
+    "homepage": "https://www.elgato.com/en/gaming/downloads#sd-win",
+    "description": "Stream Deck lets you take control of your content and focus on what's important: your audience.",
+    "version": "4.3.3.11845",
+    "license": {
```

I cannot find any URL providing URL.

```suggestion
    "license": "Freeware",
```